site/content/en/docs/Integration Patterns/player-capacity.md

```diff
@@ -7,10 +7,17 @@ description: >
   Find a `GameServer` that has room for a specific number of players.
 ---
 
+{{% feature expiryVersion="1.34.0" %}}
 {{< alpha
 title="Player Tracking and Allocation Player Filter"
 gate="PlayerTracking,PlayerAllocationFilter" >}}
 {{< beta title="Allocation State Filter" gate="StateAllocationFilter" >}}
```

Comment:
Delete this line `{{< beta title="Allocation State Filter" gate="StateAllocationFilter" >}}`

Reply:
Lines in the expiryVersion block will be removed during the release,

```
{{% feature expiryVersion="1.34.0" %}}
{{< alpha
title="Player Tracking and Allocation Player Filter"
gate="PlayerTracking,PlayerAllocationFilter" >}}
{{< beta title="Allocation State Filter" gate="StateAllocationFilter" >}}
{{% /feature %}}
{{% feature publishVersion="1.34.0" %}}
{{< alpha
title="Player Tracking and Allocation Player Filter"
gate="PlayerTracking,PlayerAllocationFilter" >}}
{{% /feature %}}
```

Reply:
Ah excellent!